mlrun/projects/project.py

```diff
@@ -2352,6 +2391,24 @@ def create_remote(self, url, name="origin", branch=None):
         self.spec._source = self.spec.source or url
         self.spec.origin_url = self.spec.origin_url or url
 
+    def remove_remote(self, name):
+        """Remove a remote from the project's Git repository.
+
+        This method removes the remote repository associated with the specified name from the project's Git repository.
+
+        :param name: Name of the remote to remove.
+        """
+        if self._remote_exists(name):
+            self.spec.repo.delete_remote(name)
+        else:
+            raise mlrun.errors.MLRunNotFoundError(
+                f"Remote '{name}' not found in the project's Git repository."
+            )
```

Comment:
Why raising an exception and not making it idempotent?
Remove a non-existent remote - it is basically removed successfully